modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-pvs-csi_{context}"]
+= Backing up persistent volumes with Container Snapshot Interface snapshots
+
+You back up persistent volumes with Container Snapshot Interface (CSI) snapshots by creating a `VolumeSnapshotClass` CR.
+
+.Prerequisites
+
+* The cloud provider must support CSI snapshots.
+* You must install the OpenShift API for Data Protection (OADP) Operator and the Data Protection Application.
+* You must add the `csi` default plugin and the `EnableCSI` feature flag to the `DataProtectionApplication` manifest:
++
+.Example DataProtectionApplication manifest with CSI
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+...
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - csi
+    featureFlags:
+    - EnableCSI
+----
+
+.Procedure
+
+. Create a `VolumeSnapshotClass` manifest for CSI, as in the following example:
++
+.Example VolumeSnapshotClass manifest
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: <csi-aws-example>
+  labels:
+    velero.io/csi-volumesnapshot-class: "true"
+driver: <csi_driver> <1>
+deletionPolicy: Retain
+----
+<1> Specify the CSI driver, for example, `ebs.csi.aws.com` for Amazon Web Services EBS.
+
+. Retrieve the `backupStorageLocations` CRs:
++
+[source,terminal]
+----
+$ oc get backupStorageLocations
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
+velero-sample-1   Available   11s              31m
+----
+
+. Create a `Backup` CR, as in the following example:
++
+[source,yaml]
+----
+$ cat << EOF | oc apply -f -
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup>
+  labels:
+    velero.io/storage-location: default
+  namespace: openshift-adp
+spec:
+  hooks: {}
+  includedNamespaces:
+  - <namespace> <1>
+  includeClusterResources: true
+  storageLocation: <velero-sample-1> <2>
+  ttl: 720h0m0s
+EOF
+----
+<1> Specify an array of namespaces to back up.
```

Comment:
Will users know how to specify an array? (I'm not sure I do, but I'm not the target audience). 
Should you add more examples to show how to configure? 

  includedNamespaces:
  - <namespace>
  - <namespace2>

Reply:
Cluster admins know how to create an array. It's just a list. We use them all the time. No need to include example.